specification/appendix/origination_of_cost_data.md

```diff
@@ -0,0 +1,25 @@
+# Origination of cost data
+
+Cost data presented in the billing datasets originates from various sources depending on the purchasing mechanism. There are at least 3 different pieces of information that are important for understanding where cost originated from.
+
+* Provider: The entity that made the infrastructure and/or services available for purchase.
+* Publisher: The entity that produced the infrastructure and/or services that were purchased
+* Invoicing Entity: The entity responsible for invoicing for the infrastructure and/or services consumed
```

Comment:
"Entity" is generic. Is there another term that's more specific? Maybe "Invoice Issuer"?
```suggestion
* Invoicing Entity: The entity responsible for invoicing for the infrastructure and/or services consumed.
```

Reply:
Updated